packages/core/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -87,6 +87,18 @@ export interface DropdownComponentProps extends CustomMenuBaseProps, CustomOptio
    * Called when blurred
    */
   onBlur?: FocusEventHandler;
+  /**
+   * Called when scrolling
+   */
+  onScroll?: (event: any) => void;
+  /**
+   * Fired when the user scrolls to the bottom of the menu
+   */
+  onMenuScrollToBottom?: (event: WheelEvent) => void;
```

Comment:
should it be:
```suggestion
  onMenuScrollToBottom?: (event: WheelEvent<HTMLDivElement>) => void;
```
?

Reply:
TS2315: Type WheelEvent is not generic.
![image](https://github.com/user-attachments/assets/7c97e84c-811c-433e-9bc6-1b47b194a2e8)

Reply:
```suggestion
  onMenuScrollToBottom?: WheelEventHandler<HTMLDivElement>;
```